.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,71 @@
+name: Bug Report
+description: I experienced unexpected behavior using useSimpleReducer.
+labels: ['bug']
+body:
+  - type: markdown
+    attributes:
+      value: Found a bug? Thanks for taking the time to report it and help make useSimpleReducer better! Fill out the form below to help our team get started.
+
+  - type: markdown
+    attributes:
+      value: |
+        ## Environment
+        What environments were you in when you encoutnered this bug?
+
+  - type: input
+    attributes:
+      label: Node
+      description: What version of Node?
+      placeholder: '16'
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: React
+      description: What version of React?
+      placeholder: '17'
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Additional Environment Information
+      description: Any other environment configurations we should be aware of?
+
+  - type: markdown
+    attributes:
+      value: |
+        ## Bug Description
+
+  - type: textarea
+    attributes:
+      label: Actual Behavior
+      description: What's the bug you're experiencing? Are there any logs or errors?
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Expected behavior
+      description: What do you expect to happen instead?
+    validations:
+      required: true
+
+  - type: markdown
+    attributes:
+      value: |
+        ## Reproduction
+
+  - type: input
+    attributes:
+      label: CodeSandbox
+      description: To expedite the process, provide a link to a CodeSandbox or Repository where this error can be reproduced. You can use [Bitovi's useSimpleReducer Template](https://codesandbox.io/s/github/bitovi/use-simple-reducer/tree/error-handling/demo) to create one quickly.
```

Comment:
specifically, we want _minimal_ reproduction sandboxes. We don't want somebody to stuff their project in, we want them to make the smallest possible app that reproduces it.